Godeps/_workspace/src/github.com/GoogleCloudPlatform/kubernetes/pkg/controller/controller_utils.go

```diff
@@ -301,6 +303,31 @@ func filterActivePods(pods []api.Pod) []*api.Pod {
 	return result
 }
 
+// filterPodsOnUID returns pods that have not terminated.
+func filterPodsOnUID(pods []*api.Pod, rcUID types.UID) []*api.Pod {
+	var result []*api.Pod
+
+	for i := range pods {
+		pod := pods[i]
+		createdByRef, ok := pod.Annotations["kubernetes.io/created-by"]
+		if !ok {
+			continue
```

Comment:
@smarterclayton was the issue that this should still be considered a match? e.g. we include pods unless we can prove they have `created-by` annotations for something that is not us?